### PR TITLE
Homework,lesson11,task1

### PR DESCRIPTION
**Pull request by mayakoff96**
@Masergio, please, review. Task 2 will be added soon.

### CODE REVIEW

**Comment by mayakoff96** [comment]
@Masergio, Task 2 was added, but I haven't seen a pull request for that one.

File: Lesson11/Task2.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    print("Контакт успішно видалений!")
+
+
+def update_contact(phone_number):
+    phonebook = load
+
+
+def update_contact(phone_number):
+    phonebook = load_phonebook()
+    for contact in phonebook:
```
Comment (Masergio) [inline Lesson11/Task2.py]: Two similar function names. Please cleanup

**Review by Masergio — COMMENTED** [review]
Good solution 👍

File: Lesson11/Task2.py (hunk 1 above)
Comment (mayakoff96) [reply Lesson11/Task2.py]: Hi, @Masergio  could you check again, I've made some changes in the code, from 124 lines (Task2.py) and Task(1) - it's an old one.

**Review by Masergio — APPROVED** [review]

**Pull request merged**